### PR TITLE
Updated values for 0.17: belts, mining, materials processing

### PR DESCRIPTION
**Pull request by s3bash**
I have re calculated and updated the values for the materials processing and mining data based on the new values in 0.17 for belt speed,crafting time, etc.

My calculations can be found here: https://docs.google.com/spreadsheets/d/1QDm0FMYdMLyXM8YXluJ400T7TOUU49LO6TKPYYYrlDY/edit?usp=sharing

### CODE REVIEW

**Comment by deniszholob** [comment]
Hey thanks for the commit, i have created a 0.17 branch blease check that one out and make the PR against it.

You should be able to check out the 0.17 branch and make a new fix branch on top of it
Then when you create a PR target that 0.17 branch instead of master.
When all of the 0.17 changes are done, ill merge the entire 0.17 branch into master.

https://factoriocheatsheet.com/
https://v016.factoriocheatsheet.com/
https://v017.factoriocheatsheet.com/

I got PR builds going too, if you click the "Details" on the "deploy/netlify" check you will be able to see the live deployment :P

If you are working on an issue thats already created, just comment on the ticket, and ill assign your name to it :)

File: src/app/views/cheat-sheets/mining/mining.component.html [hunk 1]
```diff
@@ -3,7 +3,8 @@
         <div class="col-12 col-lg-6">
             <ul>
                 <li><a href="https://wiki.factorio.com/Mining" target="_blank" rel="noopener">Mining speed</a> relies on more variables than most other machinery.</li>
-                <li>Ores have <i>mining hardness</i> and <i>mining time</i>, and miners have <i>mining power</i> and <i>mining speed</i>.</li>
+                <li>0.17 has simplified the mining calculation for the base game.</li>
```
Comment (deniszholob) [inline src/app/views/cheat-sheets/mining/mining.component.html]: I think we can get away without mentioning 0.17 changes specifically, after its standard no1 will care how it compares with previous versions

**Comment by HonkingGoose** [comment]
@deniszholob maybe you should assign @s3bash to the issues: #97 #79

**Comment by deniszholob** [comment]
Actually for some reason i only can add you...

File: src/app/views/cheat-sheets/mining/mining.component.html (hunk 1 above)
Comment (s3bash) [reply src/app/views/cheat-sheets/mining/mining.component.html]: Sounds good to me, will be removing the 0.17 line completely, and redoing the pull request to the 0.17 branch.

**Comment by HonkingGoose** [comment]
@deniszholob Hmm, maybe he needs to have a merged pull-request first? So that when he has a merged pull-request you can assign him? Maybe Github assumes that you would only want to assign to somebody who has done usefull work before on your repository?

**Comment by s3bash** [comment]
Well, I've created a new pull request to the v017 branch, hopefully did it correctly, also incorporated another text change removing the  the 0.17 version reference.

**Comment by s3bash** [comment]
Closing this pull request as it is not merging into the correct branch, as per discussion. Also feel free to assign those issues to me.

**Pull request closed without merging**